chainer/functions/__init__.py

```diff
@@ -43,6 +43,7 @@
 from chainer.functions.noise import dropout
 from chainer.functions.noise import gaussian
 from chainer.functions.normalization import batch_normalization
+from chainer.functions.normalization import l2_norm_squared
```

Comment:
I think this function should not be in `normalization` because it does not normalize vectors. A alternative directory is `math`.